content/dev-tools/utils/data/connections.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+      "id": "wstool-1-connection-s1",
```

Comment:
"wstool-1" is redundant here.  If you need a unique id you could always remove spaces and lowercase shortname.